SwiftLeeds/Views/Sponsors/SponsorsView.swift

```diff
@@ -46,6 +51,9 @@ struct SponsorsView: View {
         .scrollContentBackground(.hidden)
         .fitToReadableContentGuide(type: .width)
         .task { try? await viewModel.loadSponsors() }
+        .sheet(isPresented: $isPresentWebView) {
+            NavigationStack { WebView(url: urlToOpen) }
```

Comment:
question: What's the justification for moving towards an in-app web view as opposed to continuing to use openURL which opens in Safari?

Reply:
We should definitely still be opening jobs and sponsors in Safari

Reply:
> We should definitely still be opening jobs and sponsors in Safari

Addressed [Navigate links to Safari](https://github.com/SwiftLeeds/swiftleeds-ios/pull/44/commits/66eaee7c231d7e06a7ce30709a5703534ae6fa77)